etherman/etherman_test.go

```diff
@@ -123,6 +123,9 @@ func TestForcedBatchEvent(t *testing.T) {
 	assert.Equal(t, auth.From, blocks[0].ForcedBatches[0].Sequencer)
 }
 
+// TODO: Review tests with Joan
```

Comment:
This may be uncommented

Reply:
I try to uncomment the tests and are still failing, I'll  fix these test in v0.6.0 and cherry-picked to develop